Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1970,6 +1990,29 @@ void Element::invalidate_style_after_attribute_change(FlyString const& attribute
         return;
     }
 
+    if (attribute_name == HTML::AttributeNames::class_) {
+        Vector<StringView> old_classes;
+        Vector<StringView> new_classes;
```

Comment:
Can we go to `FlyString` directly here instead of awkwardly using `StringView`?

Reply:
those hold result of `split_view_if()` invocation on a class list and then FlyString is constructed only for added/removed class names. is there any reason to immediately convert `Vector<StringView>` to `Vector<FlyString>` instead of doing it on the go only for relevant class names?